README.md

```diff
@@ -1 +1,36 @@
 # puppet-healthcheck
+
+## Providers
+
+### provider: tcp_conn_validator
+
+`tcp_conn_validator` is used to verify that a service is listening on a given port.
+It could be used to test either a remote or a local service. It support both IPv4 and
+IPv6 connection string.
```

Comment:
As written, it supports only ip addrs not dns names, I think it's worth calling that out.